ui/src/app/shared/components/tags-input/tags-input-field.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import * as ReactForm from 'react-form';
+import {TagsInput} from './tags-input';
+
+export const TagsInputField = ReactForm.FormField((props: {options: string[]; noTagsLabel?: string; fieldApi: ReactForm.FieldApi; className: string}) => {
```

Comment:
IntelliJ tells me that this is unused. Can you check?

Reply:
It's introduced from argo-cd code base. I removed it.